docs/upgrading-2.0.md

```diff
@@ -5,6 +5,10 @@ SQLDelight 2.0 makes some breaking changes to the gradle plugin and runtime APIs
 This page lists those breaking changes and their new 2.0 equivalents. 
 For a full list of new features and other changes, see the [changelog](../changelog).
 
+The Gradle plugin (`app.cash.sqldelight`) version `2.0` isn't compatible with older versions `1.x` of the IntelliJ plugin.
+You have to update the IntelliJ plugin by adding an IntelliJ repository in the IntelliJ plugins settings, either use [alpha versions](https://plugins.jetbrains.com/plugins/alpha/com.squareup.sqldelight) or [snapshots of the master branch](https://plugins.jetbrains.com/plugins/eap/com.squareup.sqldelight).
```

Comment:
I don't think this is the right place for this notice. Eventually this page will be the "breaking changes" for the final 2.0 release so calling out the alpha and snapshot builds of the plugin there would be weird.

Reply:
We could maybe update the snapshot section on the index page to mention that newer (snapshot) versions of the gradle plugin aren't necessarily compatible with the non-snapshot IDE plugin?